### PR TITLE
app/vmctl: add support the `week` step for time-based chunks

### PR DESCRIPTION
**Pull request by dmitryk-dk**
Added support of the `week` step.

Related issue: https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4738

Signed-off-by: dmitryk-dk [d.kozlov@victoriametrics.com](mailto:d.kozlov@victoriametrics.com)

### CODE REVIEW

**Review by f41gh7 — APPROVED** [review]
LGTM

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ The following `tip` changes can be tested by building VictoriaMetrics components
 ## tip
 
 * BUGFIX: [vmagent](https://docs.victoriametrics.com/vmagent.html): use local scrape timestamps for the scraped metrics unless `honor_timestamps: true` option is explicitly set at [scrape_config](https://docs.victoriametrics.com/sd_configs.html#scrape_configs). This fixes gaps for metrics collected from [cadvisor](https://github.com/google/cadvisor) or similar exporters, which export metrics with invalid timestamps. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4697) and [this comment](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4697#issuecomment-1654614799) for details. The issue has been introduced in [v1.68.0](#v1680).
+* FEATURE: [vmctl](https://docs.victoriametrics.com/vmctl.html): add support `week` step for time-based chunks. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4738).
```
Comment (hagen1778) [inline docs/CHANGELOG.md:28]:
```suggestion
* FEATURE: [vmctl](https://docs.victoriametrics.com/vmctl.html): add support of `week` step for [time-based chunking migration](https://docs.victoriametrics.com/vmctl.html#using-time-based-chunking-of-migration). See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4738).
```

**Review by hagen1778 — APPROVED** [review]
LGTM

**Comment by valyala** [comment]
@dmitryk-dk , FYI, see the follow-up commit, which adds missing documentation for the `week` value - f96804658bac738cdf7d70e92926c8fe1e372e4d

**Pull request merged**